types/electra/attestation.yaml

```diff
@@ -0,0 +1,33 @@
+Electra:
+  IndexedAttestation:
+    type: object
+    description: "The [`IndexedAttestation`](https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/beacon-chain.md#indexedattestation) object from the CL spec."
+    properties:
+      attesting_indices:
+        type: array
+        maxItems: 131072
+        description: "Attesting validator indices"
+        items:
+          $ref: "../primitive.yaml#/Uint64"
+      signature:
+        $ref: '../primitive.yaml#/Signature'
+        description: "The BLS signature of the `IndexedAttestation`, created by the validator of the attestation."
+      data:
+        $ref: '../attestation_data.yaml#/AttestationData'
+
+  Attestation:
```

Comment:
this conflicts with the other open PR https://github.com/ethereum/beacon-APIs/pull/447, might wanna get consensus there on how we go about updating all the APIs and then rebase this branch

Reply:
Absolutely, I had in mind to rebase this one once the #447 gets merged